docs/src/pages/[platform]/components/avatar/demo.tsx

```diff
@@ -0,0 +1,43 @@
+import * as React from 'react';
+import { Avatar, AvatarProps } from '@aws-amplify/ui-react';
+
+import { AvatarPropControls } from './AvatarPropControls';
+import { useAvatarProps } from './useAvatarProps';
+import { Demo } from '@/components/Demo';
+import { demoState } from '@/utils/demoState';
+
+const propsToCode = (avatarProps: AvatarProps) => {
+  const { size, variation, colorTheme, src } = avatarProps;
+  return (
+    `<Avatar` +
+    (src ? `\n  src=${JSON.stringify(src)}` : '') +
+    (size ? `\n  size=${JSON.stringify(size)}` : '') +
+    (colorTheme ? `\n  colorTheme=${JSON.stringify(colorTheme)}` : '') +
+    (variation ? `\n  variation=${JSON.stringify(variation)}` : '') +
+    `\n/>`
+  );
+};
+
+const defaultAvatarProps = {
+  src: '/cats/1.jpg',
```

Comment:
Just a nit: should we use a transparent avatar or svg icon so the demo can show the variations better? Currently the demo doesn't make it clear that filled/default variations do much when you change them.